app/access/api.py

```diff
@@ -1,6 +1,9 @@
+from cognito.utils.user import create_user
```

Comment:
Maybe rename this function to `create_cognito_user` so it's more clear in the code below. See comment there.